codec/src/main/java/io/netty/handler/codec/LengthFieldPrepender.java

```diff
@@ -116,6 +119,28 @@ public LengthFieldPrepender(int lengthFieldLength, int lengthAdjustment) {
      *         if {@code lengthFieldLength} is not 1, 2, 3, 4, or 8
      */
     public LengthFieldPrepender(int lengthFieldLength, int lengthAdjustment, boolean lengthIncludesLengthFieldLength) {
+        this(ByteOrder.nativeOrder(), lengthFieldLength, lengthAdjustment, lengthIncludesLengthFieldLength);
```

Comment:
I think we should just use ByteOrder.BIG_ENDIAN as this what we also use as default in netty when you create a buffer.